scripts/centralize_database

```diff
@@ -6,19 +6,19 @@ import redis
 import argparse
 
 def centralize_to_target_db(target_dbname):
-    target_dbport = SonicDBConfig.get_port(target_dbname)
-    target_dbhost = SonicDBConfig.get_hostname(target_dbname)
+    target_dbport = SonicDBConfig.getDbPort(target_dbname)
```

Comment:
>getDbPort [](start = 34, length = 9)

Thanks for the fixes! They are valid. There is another way to use swig to translate the names by a single wrapper like [https://github.com/Azure/sonic-swss-common/blob/9e91e0d891398b468b8087682ae91335791fac51/common/dbconnector.h#L76](https://github.com/Azure/sonic-swss-common/blob/9e91e0d891398b468b8087682ae91335791fac51/common/dbconnector.h#L76).

If the functions are used in multiple repos, the renaming will be better.

Reply:
I see, thanks for the info!
I was thinking that we are planning to deprecate the Python APIs, and want to stick to C++ library.
If that is correct understanding, then shouldn't we move away from using SWIG wrappers?

> If the functions are used in multiple repos, the renaming will be better.
In this case, we should have similar change as in this PR for other repos too. Basically, to maintain C++ method names, and less of translated Python methods.

Thoughts?

Reply:
It's a trade-off: Adding SWIG wrappers means more code to maintain, but allows us to align the naming convention with PEP8, where function/method names should be `lowercase_with_underscores`, whereas in C++ our function names are camel case.